homeassistant/components/ping/ping.py

```diff
@@ -0,0 +1,162 @@
+"""Ping classes shared between platforms."""
+import asyncio
+from contextlib import suppress
+import logging
+import re
+from typing import TYPE_CHECKING, Any
```

Comment:
The change looks fine, but can you pick a name that better shows intent besides ping.py?

Reply:
Yes, sure, renamed it.